notebooks/book1/04/samplingDistributionGaussianShrinkage.ipynb

```diff
@@ -0,0 +1,577 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        logp = -0.5 * jnp.sum(X * X / Sigma, axis=1)
</pre><p>Just curious -- would it make sense to use tfp.distributions and get the logprob through that? It becomes easier to comprehend the code that way. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>